src/pylyrics/clean_text.py

```diff
@@ -0,0 +1,40 @@
+# Authors: Abhiket Gaurav, Artan Zandian, Macy Chan, Manju Abhinandana Kumar
+# January 2022
+
+import re
+from pylyrics import extract_lyrics
+
+
+def clean_text(text):
+    """Cleans the text by removing special characters, html_tags, #tags, contaction words.
```

Comment:
Also output as lower case

Reply:
Please also move this to another brand and remove this file here, otherwise will have conflict